libvast/src/system/partition.cpp

```diff
@@ -124,15 +124,21 @@ caf::actor fetch_indexer(table_indexer& tbl, const attribute_extractor& ex,
       return [=](const curried_predicate&) { return row_ids; };
     });
   }
+  // Predicate of form "&time == ..."
```

Comment:
Is this really always an equality? It seems the `if` statement below is enough to tell the user that we're dealing with a `&time` extractor.